src/test/data/JsonTutorsPetStorageTest/invalidStudentTutorsPet.json

```diff
@@ -4,5 +4,6 @@
     "name": "Student with invalid name field: Ha!ns Mu@ster",
     "telegram": "hans_M4STERZ",
     "email": "hans@example.com"
-  } ]
+  } ],
+  "classes" : [ ]
```

Comment:
Should there be at least one class entry here? We can follow `invalidClassTutorsPet.json`, which has a valid student entry.

Reply:
I initially added a class entry, but then eventually removed them as I wanted to isolate the test cases better (Since this json file is more to test for invalidstudent rather than invalidclass. Dexter, any thoughts?

Reply:
Resolved by keeping status quo for invalidClassTutorsPet.json (ie no changes)